annif/project.py

```diff
@@ -178,6 +201,8 @@ def suggest(self, text, backend_params=None):
                 raise NotInitializedException('Project is not trained.')
         logger.debug('Suggesting subjects for text "%s..." (len=%d)',
                      text[:20], len(text))
+        if self.transformer is not None:
```

Comment:
If the property function above falls back to a passthrough transformer (as suggested above), then there is no need for the None check here.